src/pal/src/misc/tracepointprovider.cpp

```diff
@@ -59,6 +59,14 @@ __attribute__((constructor (200)))
 static void
 PAL_InitializeTracing(void)
 {
+    int shouldDisableLoad = 0;
+    // Check if loading the LTTng providers should be disabled.
+    char *disableValue = getenv("COMPlus_DisableLttngLibraryLoad");
```

Comment:
Nit: It is better to avoid `Disable` in the name. It makes the name long and conceptually harder to think about because of you need to think about double-negation. 

Can this be called just `COMPlus_Lttng` ?

Reply:
How about `COMPlus_LttngOptOut`, since the default behavior is for LTTng to be on?

Reply:
It has the same problem with double negation. The default value of the switch should not affect the naming. We have number of features that are on by default and that you set to `0` to disable. For example:

`COMPlus_ReadyToRun=0`
`COMPlus_TieredCompilation=0`
`COMPlus_TC_QuickJit=0`

Also, the defaults change over time.

Reply:
Sounds good.  I can use `COMPlus_Lttng`.